adapter/stackdriver/BUILD

```diff
@@ -1,52 +1,30 @@
-package(default_visibility = ["//visibility:public"])
-
-load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_prefix", "go_test")
 
 go_library(
     name = "go_default_library",
-    srcs = [
-        "bufferedClient.go",
-        "merge.go",
-        "stackdriver.go",
-    ],
+    srcs = ["stackdriver.go"],
+    visibility = ["//visibility:public"],
     deps = [
         "//adapter/stackdriver/config:go_default_library",
+        "//adapter/stackdriver/log:go_default_library",
+        "//adapter/stackdriver/metric:go_default_library",
         "//pkg/adapter:go_default_library",
+        "//template/logentry:go_default_library",
         "//template/metric:go_default_library",
-        "@com_github_golang_protobuf//ptypes:go_default_library",
-        "@com_github_golang_protobuf//ptypes/timestamp:go_default_library",
-        "@com_github_googleapis_gax_go//:go_default_library",
-        "@com_github_istio_api//:mixer/v1/config/descriptor",
-        "@com_google_cloud_go//monitoring/apiv3:go_default_library",
-        "@org_golang_google_api//option:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/label:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/metric:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/monitoredres:go_default_library",
-        "@org_golang_google_genproto//googleapis/monitoring/v3:go_default_library",
-        "@org_golang_x_net//context:go_default_library",
+        "@com_github_hashicorp_go_multierror//:go_default_library",
     ],
 )
 
 go_test(
     name = "go_default_test",
-    srcs = [
-        "bufferedClient_test.go",
-        "merge_test.go",
-        "stackdriver_test.go",
-    ],
+    srcs = ["stackdriver_test.go"],
     library = ":go_default_library",
     deps = [
-        "//adapter/stackdriver/config:go_default_library",
+        "//pkg/adapter:go_default_library",
         "//pkg/adapter/test:go_default_library",
+        "//template/logentry:go_default_library",
         "//template/metric:go_default_library",
-        "@com_github_golang_protobuf//ptypes:go_default_library",
-        "@com_github_golang_protobuf//ptypes/timestamp:go_default_library",
-        "@com_github_googleapis_gax_go//:go_default_library",
-        "@com_google_cloud_go//monitoring/apiv3:go_default_library",
-        "@org_golang_google_api//option:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/metric:go_default_library",
-        "@org_golang_google_genproto//googleapis/api/monitoredres:go_default_library",
-        "@org_golang_google_genproto//googleapis/monitoring/v3:go_default_library",
-        "@org_golang_x_net//context:go_default_library",
     ],
 )
+
+go_prefix("istio.io/mixer")
```

Comment:
curious: why add `go_prefix` here?

Reply:
gazelle adds it

Reply:
It's weird gazelle adds go_prefix here. Did you run `./bin/gazelle` script or run the gazelle command by yourself?

Reply:
Ran gazelle directly myself, though IIRC the script adds it too. Removed it either way.

Reply:
thanks. I will look into why gazelle runs weirdly after this PR is merged.